statsdaemon.go

```diff
@@ -471,7 +472,7 @@ func parseMessage(data []byte) []*Packet {
 		}
 
 		packet := &Packet{
-			Bucket:   sanitizeBucket(*prefix + string(name)),
+			Bucket:   sanitizeBucket(*prefix + string(name) + *postfix),
```

Comment:
when I tried to do this, the bucket names for timings get wrong, the postfix gets added before the percentile, mean etc.